cves/CVE-2016-3087.yml

```diff
@@ -144,12 +156,19 @@ interesting_commits:
     emerging themes?
     If there are no interesting commits, demonstrate that you completed this section
     by explaining what happened between the VCCs and the fix.
-  answer:
+  answer: |
+    Git log only returned two commits betwwen the vcc and the fix. The latter
```

Comment:
Misspelling for the word between. I also think that using the terms latter and former can be confusing to some people who aren't too familiar with that kind of speech.